main.py

```diff
@@ -233,6 +233,26 @@ def upload_form():
            <p><a href="{home}">Return home</a></p>
            """.format(home=url_for('home'))
 
+@app.route('/upload/<upload_id>', methods=['GET'])
+def update_status(upload_id):
+    """
+    Shows the status of an upload.
+    :param upload_id:
+    :return:
```

Comment:
looking at your revisions.. sorry for not being clear. i only meant that the params and return were undocumented.